nvidia_deeplearningexamples_gpunet.md

```diff
@@ -0,0 +1,133 @@
+
+The model architecture is visible as output of the loaded model. For details architecture and latency info please refer to Figure#[6](https://drive.google.com/file/d/1pLA0OopYVRpYreIhQQKPdNauaaBEGITe/view) and Table#[3](https://drive.google.com/file/d/1pLA0OopYVRpYreIhQQKPdNauaaBEGITe/view) in the CVPR-2022 paper. 
+
+Please pick and choose one of the following pre-trained models:
+
+| TorchHub model | Description |
+| :----- | :----- |
+| `GPUNet-0` | GPUNet-0 has the fastest measured latency on GV100 |
+| `GPUNet-1` | GPUNet-1 has improved accuracy with one additional layer on GPUNet-0|
+| `GPUNet-2` | GPUNet-2 has higher accuracy with two additional layers on GPUNet-0 |
+| `GPUNet-P0` | GPUNet-P0 is the distilled model with higher accuracy than GPUNet-0 but similar latency|
+| `GPUNet-P1` | GPUNet-P1 is distilled model with even higher accuracy than GPUNet-1 but similar latency |
+| `GPUNet-D1` | GPUNet-D1 has the second highest accuracy amongst all GPUNets|
+| `GPUNet-D2` | GPUNet-D2 has the highest accuracy amongst all GPUNets |
+
+***Note***: Loaded model defaults to fp32 weights. To switch to fp16 gpunet model weights, set parameter ***model_math='fp16'***.
+```python
+gpunet = torch.hub.load('NVIDIA/DeepLearningExamples:torchhub', 'nvidia_gpunet', pretrained=True, model_type='GPUNet-0')
+utils = torch.hub.load('NVIDIA/DeepLearningExamples:torchhub', 'nvidia_convnets_processing_utils')
+
+gpunet.eval().to(device)
+
+```
+
+### Prepare inference data
+Prepare sample input data for inference.
+```python
+uris = [
+    'http://images.cocodataset.org/test-stuff2017/000000024309.jpg',
+    'http://images.cocodataset.org/test-stuff2017/000000028117.jpg',
+    'http://images.cocodataset.org/test-stuff2017/000000006149.jpg',
+    'http://images.cocodataset.org/test-stuff2017/000000004954.jpg',
+]
+
+batch = torch.cat(
+    [utils.prepare_input_from_uri(uri) for uri in uris]
+).to(device)
+
+print("Ready to run inference...")
+```
+
+### Run inference
+Use `pick_n_best(predictions=output, n=topN)` helper function to pick N most probable hypotheses according to the model.
+
+***Note***: If using fp16 gpunet model weights replace gpunet(batch) with ***gpunet(batch.half())***.
+```python
+with torch.no_grad():
+    output = torch.nn.functional.softmax(gpunet(batch), dim=1)
+    
+results = utils.pick_n_best(predictions=output, n=5)
+```
+
+### Display result
+```python
+for uri, result in zip(uris, results):
+    img = Image.open(requests.get(uri, stream=True).raw)
+    img.thumbnail((256,256), Image.ANTIALIAS)
+    plt.imshow(img)
+    plt.show()
+    print(result)
+```
+
+### Details
+For detailed information on model input and output, training recipies, inference and performance visit:
+[github](https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/Classification/GPUNets)
```

Comment:
the link should be https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/Classification/GPUNet I think

Reply:
Fixed